server/verifiers/gplus-verifier/verifier/src/main/java/org/kaaproject/kaa/server/verifiers/gplus/verifier/GplusUserVerifier.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright 2014 CyberVision, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kaaproject.kaa.server.verifiers.gplus.verifier;
+
+import org.codehaus.jackson.map.ObjectMapper;
+import org.kaaproject.kaa.server.common.verifier.AbstractKaaUserVerifier;
+import org.kaaproject.kaa.server.common.verifier.UserVerifierCallback;
+import org.kaaproject.kaa.server.common.verifier.UserVerifierContext;
+import org.kaaproject.kaa.server.verifiers.gplus.config.gen.GplusAvroConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+import java.net.*;
+import java.util.Map;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.LinkedBlockingQueue;
+import java.util.concurrent.ThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+
+public class GplusUserVerifier extends AbstractKaaUserVerifier<GplusAvroConfig> {
+    private static final Logger LOG = LoggerFactory.getLogger(GplusUserVerifier.class);
+    private static final String GOOGLE_OAUTH = "https://www.googleapis.com/oauth2/v1/tokeninfo?access_token=";
```

Comment:
I don`t really know if schema is the right place for this constant. so I`ve left it like it is, as you`ve told the first time